src/info.js

```diff
@@ -0,0 +1,228 @@
+
+    return {
+        key: 'Cordova Packages',
+        children: [{
+            key: 'cli',
+            value: cliPkg.version,
+            children: cliDependencies
+        }]
+    };
+}
+
+async function getInstalledPlatforms (projectRoot) {
+    return _getInstalledPlatforms(projectRoot).then(platforms => {
+        const key = 'Project Installed Platforms';
+        const children = Object.entries(platforms)
+            .map(([key, value]) => ({ key, value }));
+
+        return { key, children };
+    });
+}
+
+async function getInstalledPlugins (projectRoot) {
+    const key = 'Project Installed Plugins';
+    const children = cdvPluginUtil.getInstalledPlugins(projectRoot)
+        .map(plugin => ({ key: plugin.id, value: plugin.version }));
+
+    return { key, children };
+}
+
+async function getEnvironmentInfo () {
+    const [npmVersion, osInfoResult] = await Promise.all([_getNpmVersion(), osInfo()]);
+    const { platform, distro, release, codename, kernel, arch, build } = osInfoResult;
+
+    const optionalBuildSuffix = build ? ` (${build})` : '';
+
+    const osFormat = [
+        platform === 'darwin' ? codename : distro,
+        release + optionalBuildSuffix,
+        build ? `(${build})` : '',
+        `(${platform} ${kernel})`,
+        `${arch}`
+    ];
+
+    return {
+        key: 'Environment',
+        children: [
+            { key: 'OS', value: osFormat.join(' ') },
+            { key: 'Node', value: process.version },
+            { key: 'npm', value: npmVersion }
+        ]
+    };
+}
+
+async function getPlatformEnvironmentData (projectRoot) {
+    const installedPlatforms = await _getInstalledPlatforms(projectRoot);
+
+    return Object.keys(installedPlatforms)
+        .map(platform => {
+            const platformApi = getPlatformApi(platform);
+
+            const getPlatformInfo = platformApi && platformApi.getEnvironmentInfo
+                ? () => platformApi.getEnvironmentInfo()
+                : _legacyPlatformInfo[platform];
+
+            return { platform, getPlatformInfo };
+        })
+        .filter(o => o.getPlatformInfo)
+        .map(async ({ platform, getPlatformInfo }) => ({
+            key: `${platform} Environment`,
+            children: await getPlatformInfo()
+        }));
+}
+
+async function getProjectSettingsFiles (projectRoot) {
+    const cfgXml = _fetchFileContents(cdvLibUtil.projectConfig(projectRoot));
+
+    // Create package.json snippet
+    const pkgJson = require(path.join(projectRoot, 'package'));
+    const pkgSnippet = [
+        '--- Start of Cordova JSON Snippet ---',
+        JSON.stringify(pkgJson.cordova, null, 2),
+        '--- End of Cordova JSON Snippet ---'
+    ].join('\n');
+
+    return {
+        key: 'Project Setting Files',
+        children: [
+            { key: 'config.xml', value: `${cfgXml}` },
+            { key: 'package.json', value: pkgSnippet }
+        ],
+        options: {
+            addNewLineSep: true
+        }
```

Comment:
```suggestion
        ]
```

Sorry, forgot to remove these obsolete options. All multi-line values are padded with newlines now.